gradle-conjure/build.gradle

```diff
@@ -47,7 +48,46 @@ dependencies {
 }
 
 gradlePlugin {
-    automatedPublishing = false
+    plugins {
+        conjure {
+            id = 'com.palantir.conjure'
+            implementationClass = 'com.palantir.gradle.conjure.ConjurePlugin'
+        }
+        conjurePublish {
+            id = 'com.palantir.conjure-publish'
+            implementationClass = 'com.palantir.gradle.conjure.ConjurePublishPlugin'
+        }
+        conjureLocal {
+            id = 'com.palantir.conjure-local'
+            implementationClass = 'com.palantir.gradle.conjure.ConjureLocalPlugin'
+        }
+        conjureJavaLocal {
+            id = 'com.palantir.conjure-java-local'
+            implementationClass = 'com.palantir.gradle.conjure.ConjureJavaLocalCodegenPlugin'
```

Comment:
The plugin info metaresources can be deleted when these are present

Reply:
will do